src/otx/core/model/entity/detection.py

```diff
@@ -351,6 +355,39 @@ def _setup_tiler(self) -> None:
                 and overlap: {self.model.tiles_overlap}",
         )
 
+    def _create_model(self) -> Model:
+        """Create a OV model with help of Model API."""
+        from openvino.model_api.adapters import OpenvinoAdapter, create_core, get_user_config
+
+        plugin_config = get_user_config("AUTO", str(self.num_requests), "AUTO")
+        if self.use_throughput_mode:
+            plugin_config["PERFORMANCE_HINT"] = "THROUGHPUT"
+
+        model_adapter = OpenvinoAdapter(
+            create_core(),
+            self.model_name,
+            max_num_requests=self.num_requests,
+            plugin_config=plugin_config,
+            model_parameters=self.model_adapter_parameters,
+        )
+        for name, info in model_adapter.model.rt_info["model_info"].items():
+            if name == "test_meta_info":
+                for key, value in info.items():
+                    self.test_meta_info[key] = self._change_dtype(value.value)
+        return Model.create_model(model_adapter, model_type=self.model_type, configuration=self.model_api_configuration)
+
+    @staticmethod
+    def _change_dtype(value: str) -> str | bool | int | float:
+        if value == "NO":
+            return False
+        if value == "YES":
+            return True
+        if value.isdigit():
+            return int(value)
+        if value.replace(".", "", 1).isdigit():
+            return float(value)
+        return value
```

Comment:
Is it impossible just to serialize and deserialize with JSON?

Reply:
Do you mean saving export model to json file not xml file?

Reply:
No it's like
```python
("model_info", "test_meta_info"): json.dumps(self.test_meta_info)
```

Reply:
Done by 94b23d5, Thanks